src/api/test/unit/code_quality_test.rb

```diff
@@ -77,7 +77,7 @@ def setup
       'BsRequestActionSubmit#execute_accept'                                    => 126.42,
       'BsRequestPermissionCheck#cmd_changestate_permissions'                    => 117.09,
       'RequestSourceDiff::ActionSourceDiffer#diff_for_source'                   => 94.62,
-      'BuildController#file'                                                    => 127.42,
+      'BuildController#file'                                                    => 130.44,
```

Comment:
> do not ignore it - quite easy

If it is so easy then why did you ignore the complexity of `BuildController#file` in the first place? :)

https://github.com/openSUSE/open-build-service/commit/b64e8a5aa20189f91c740dca61288a1fa8f9c784#diff-1722a7f55bc92d92c2b65ff19964cd70R93